src/System.IO.FileSystem.Primitives/tests/System.IO.FileSystem.Primitives.Tests.builds

```diff
@@ -4,7 +4,11 @@
   <ItemGroup>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj"/>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj">
-      <TestTFMs>netcore50;net46</TestTFMs>
```

Comment:
Which would mean you still need a netstandard1.3 build configuration for this project.